frontend/src/lib/components/Popover.svelte

```diff
@@ -9,28 +9,16 @@
 	export let disapperTimoout = 100
 
 	const [popperRef, popperContent] = createPopperActions({ placement })
-	const betterPreventOverflow = (options) => ({
-		name: 'preventOverflow',
-		options,
-		effect: ({ state }) => {
-			const { padding = 0 } = options
 
-			state.elements.popper.style.maxWidth = `calc(100vw - ${padding * 2}px)`
-		}
-	})
-	const extraOpts = {
+	const popperOptions: PopperOptions<{}> = {
+		placement: 'bottom-end',
+		strategy: 'fixed',
```

Comment:
@rubenfiszel Just changed from absolute to fixed, https://popper.js.org/docs/v2/constructors/#strategy